python/ray/rllib/policy_gradient/policy_gradient.py

```diff
@@ -170,26 +179,33 @@ def train(self):
             batch_index = 0
             num_batches = (
                 int(tuples_per_device) // int(model.per_device_batch_size))
-            loss, kl, entropy = [], [], []
+            loss, policyloss, vfloss, kl, entropy = [], [], [], [], []
```

Comment:
`policyloss` -> `policy_loss` (including as part of other variable names like `batch_policyloss`)